src/Core/Managed/Shared/Extensibility/TelemetryConfiguration.cs

```diff
@@ -22,13 +22,85 @@ public sealed class TelemetryConfiguration : IDisposable
         private static TelemetryConfiguration active;
 
         private readonly SnapshottingList<ITelemetryInitializer> telemetryInitializers = new SnapshottingList<ITelemetryInitializer>();
+        private ITelemetryChannel telemetryChannel = null;
         private TelemetryProcessorChain telemetryProcessorChain;
         private string instrumentationKey = string.Empty;
         private bool disableTelemetry = false;
         private TelemetryProcessorChainBuilder builder;
         private SnapshottingList<IMetricProcessor> metricProcessors = new SnapshottingList<IMetricProcessor>();
 
         /// <summary>
+        /// Indicates if this instance has been disposed of.
+        /// </summary>
```

Comment:
It makes much nicer docs to read when you generate them.
And for some learning types it helps when you are new to the code and trying to figure out what things are.